gatherers/accessibility.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* global document, window, __dirname */
+
+const Gather = require('./gather');
+const fs = require('fs');
+const path = require('path');
+const axe = fs.readFileSync(path.join(__dirname, '../node_modules/axe-core/axe.min.js'));
```

Comment:
I've been bitten before by trying to predict location. modules were siblings or nested, etc.

 the `resolve` npm module sorted it out.